API Docs: Update links, copy edit, apply style guide (#1486)
slackapi/java-slack-sdk: docs/content/guides/getting-started-with-bolt.md

**Using `src/main/java/hello/MyApp.java`**

<Tabs groupId="socket-or-http">
<TabItem value="socket-mode" label="Socket Mode">

> Only single source code is required to run your first Bolt app. All you need to do is define the main method that starts `SocketModeApp`.

Reviewer: ```suggestion
Only single source code is required to run your first Bolt app. You'll need to define the main method that starts `SocketModeApp`.
```

}
```

> If you go with JDK 10+, thanks to [Local Variable Type Inference](https://docs.oracle.com/en/java/javase/21/language/local-variable-type-inference.html), your code could be much more concise. To take advantage of it, install OpenJDK 11 and set the compatible Java versions in `build.gradle` as below. Also, configure the same on your IDE.

Reviewer: ```suggestion
If you go with JDK 10+, thanks to [Local Variable Type Inference](https://docs.oracle.com/en/java/javase/21/language/local-variable-type-inference.html), you can write more concise code. To do so, install OpenJDK 11 and set the compatible Java versions in `build.gradle` as below, configuring the same on your IDE.
```
